### ISSUE TITLE
2025-03-03, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#100]
# Today's Arxiv Papers

(1) [LLM Post-Training: A Deep Dive into Reasoning Large Language Models](http://arxiv.org/abs/2502.21321v1), 2025-02-28

(2) [FANformer: Improving Large Language Models Through Effective Periodicity Modeling](http://arxiv.org/abs/2502.21309v1), 2025-02-28

(3) [Contextualizing biological perturbation experiments through language](http://arxiv.org/abs/2502.21290v1), 2025-02-28

(4) [Adaptive Keyframe Sampling for Long Video Understanding](http://arxiv.org/abs/2502.21271v1), 2025-02-28

(5) [Token-level Ensembling of Models with Different Vocabularies](http://arxiv.org/abs/2502.21265v1), 2025-02-28

(6) [RoboBrain: A Unified Brain Model for Robotic Manipulation from Abstract to Concrete](http://arxiv.org/abs/2502.21257v1), 2025-02-28

(7) [Semantic Volume: Quantifying and Detecting both External and Internal Uncertainty in LLMs](http://arxiv.org/abs/2502.21239v1), 2025-02-28

(8) [Transforming Tuberculosis Care: Optimizing Large Language Models For Enhanced Clinician-Patient Communication](http://arxiv.org/abs/2502.21236v1), 2025-02-28

(9) [ByteScale: Efficient Scaling of LLM Training with a 2048K Context Length on More Than 12,000 GPUs](http://arxiv.org/abs/2502.21231v1), 2025-02-28

(10) [ECLeKTic: a Novel Challenge Set for Evaluation of Cross-Lingual Knowledge Transfer](http://arxiv.org/abs/2502.21228v1), 2025-02-28

(11) [Transformers Learn to Implement Multi-step Gradient Descent with Chain of Thought](http://arxiv.org/abs/2502.21212v1), 2025-02-28

(12) [ARIES: Autonomous Reasoning with LLMs on Interactive Thought Graph Environments](http://arxiv.org/abs/2502.21208v1), 2025-02-28

(13) [Chronologically Consistent Large Language Models](http://arxiv.org/abs/2502.21206v1), 2025-02-28

(14) [Optimizing Large Language Models for ESG Activity Detection in Financial Texts](http://arxiv.org/abs/2502.21112v1), 2025-02-28

(15) [Large Language Model-Based Benchmarking Experiment Settings for Evolutionary Multi-Objective Optimization](http://arxiv.org/abs/2502.21108v1), 2025-02-28

(16) [Generating patient cohorts from electronic health records using two-step retrieval-augmented text-to-SQL generation](http://arxiv.org/abs/2502.21107v1), 2025-02-28

(17) [A Non-contrast Head CT Foundation Model for Comprehensive Neuro-Trauma Triage](http://arxiv.org/abs/2502.21106v1), 2025-02-28

(18) [Re-evaluating Theory of Mind evaluation in large language models](http://arxiv.org/abs/2502.21098v1), 2025-02-28

(19) [An LLM-based Delphi Study to Predict GenAI Evolution](http://arxiv.org/abs/2502.21092v1), 2025-02-28

(20) [PASemiQA: Plan-Assisted Agent for Question Answering on Semi-Structured Data with Text and Relational Information](http://arxiv.org/abs/2502.21087v1), 2025-02-28

(21) [CODI: Compressing Chain-of-Thought into Continuous Space via Self-Distillation](http://arxiv.org/abs/2502.21074v1), 2025-02-28

(22) [GUIDE: LLM-Driven GUI Generation Decomposition for Automated Prototyping](http://arxiv.org/abs/2502.21068v1), 2025-02-28

(23) [FC-Attack: Jailbreaking Large Vision-Language Models via Auto-Generated Flowcharts](http://arxiv.org/abs/2502.21059v1), 2025-02-28

(24) [The amplifier effect of artificial agents in social contagion](http://arxiv.org/abs/2502.21037v1), 2025-02-28

(25) [Beyond Words: A Latent Memory Approach to Internal Reasoning in LLMs](http://arxiv.org/abs/2502.21030v1), 2025-02-28

(26) [Measuring and identifying factors of individuals' trust in Large Language Models](http://arxiv.org/abs/2502.21028v1), 2025-02-28

(27) [Artemis: Toward Accurate Detection of Server-Side Request Forgeries through LLM-Assisted Inter-Procedural Path-Sensitive Taint Analysis](http://dx.doi.org/10.1145/3720488), 2025-02-28

(28) [PersuasiveToM: A Benchmark for Evaluating Machine Theory of Mind in Persuasive Dialogues](http://arxiv.org/abs/2502.21017v1), 2025-02-28

(29) [Explainable Biomedical Claim Verification with Large Language Models](http://arxiv.org/abs/2502.21014v1), 2025-02-28

(30) [Merging Clinical Knowledge into Large Language Models for Medical Research and Applications: A Survey](http://arxiv.org/abs/2502.20988v1), 2025-02-28

(31) [UoR-NCL at SemEval-2025 Task 1: Using Generative LLMs and CLIP Models for Multilingual Multimodal Idiomaticity Representation](http://arxiv.org/abs/2502.20984v1), 2025-02-28

(32) [Set-Theoretic Compositionality of Sentence Embeddings](http://arxiv.org/abs/2502.20975v1), 2025-02-28

(33) [Arabizi vs LLMs: Can the Genie Understand the Language of Aladdin?](http://arxiv.org/abs/2502.20973v1), 2025-02-28

(34) [TeleRAG: Efficient Retrieval-Augmented Generation Inference with Lookahead Retrieval](http://arxiv.org/abs/2502.20969v1), 2025-02-28

(35) [Beware of Your Po! Measuring and Mitigating AI Safety Risks in Role-Play Fine-Tuning of LLMs](http://arxiv.org/abs/2502.20968v1), 2025-02-28

(36) [Fine-Grained Retrieval-Augmented Generation for Visual Question Answering](http://arxiv.org/abs/2502.20964v1), 2025-02-28

(37) [Retrieval Augmented Generation for Topic Modeling in Organizational Research: An Introduction with Empirical Demonstration](http://arxiv.org/abs/2502.20963v1), 2025-02-28

(38) [Efficient Jailbreaking of Large Models by Freeze Training: Lower Layers Exhibit Greater Sensitivity to Harmful Content](http://arxiv.org/abs/2502.20952v1), 2025-02-28

(39) [A Deep User Interface for Exploring LLaMa](http://arxiv.org/abs/2502.20938v1), 2025-02-28

(40) [WebFAQ: A Multilingual Collection of Natural Q&A Datasets for Dense Retrieval](http://arxiv.org/abs/2502.20936v1), 2025-02-28

(41) [Large Language Models Are Innate Crystal Structure Generators](http://arxiv.org/abs/2502.20933v1), 2025-02-28

(42) [Automated Evaluation of Meter and Rhyme in Russian Generative and Human-Authored Poetry](http://arxiv.org/abs/2502.20931v1), 2025-02-28

(43) [A database to support the evaluation of gender biases in GPT-4o output](http://arxiv.org/abs/2502.20898v1), 2025-02-28

(44) [Beyond Demographics: Fine-tuning Large Language Models to Predict Individuals' Subjective Text Perceptions](http://arxiv.org/abs/2502.20897v1), 2025-02-28

(45) [PathVG: A New Benchmark and Dataset for Pathology Visual Grounding](http://arxiv.org/abs/2502.20869v1), 2025-02-28

(46) [ProBench: Benchmarking Large Language Models in Competitive Programming](http://arxiv.org/abs/2502.20868v1), 2025-02-28

(47) [Better Benchmarking LLMs for Zero-Shot Dependency Parsing](http://arxiv.org/abs/2502.20866v1), 2025-02-28

(48) [The Power of Personality: A Human Simulation Perspective to Investigate Large Language Model Agents](http://arxiv.org/abs/2502.20859v1), 2025-02-28

(49) [A Pilot Empirical Study on When and How to Use Knowledge Graphs as Retrieval Augmented Generation](http://arxiv.org/abs/2502.20854v1), 2025-02-28

(50) [Learning to Substitute Components for Compositional Generalization](http://arxiv.org/abs/2502.20834v1), 2025-02-28

(51) [CoTMR: Chain-of-Thought Multi-Scale Reasoning for Training-Free Zero-Shot Composed Image Retrieval](http://arxiv.org/abs/2502.20826v1), 2025-02-28

(52) [LADs: Leveraging LLMs for AI-Driven DevOps](http://arxiv.org/abs/2502.20825v1), 2025-02-28

(53) [Towards Reliable Vector Database Management Systems: A Software Testing Roadmap for 2030](http://arxiv.org/abs/2502.20812v1), 2025-02-28

(54) [HAIC: Improving Human Action Understanding and Generation with Better Captions for Multi-modal Large Language Models](http://arxiv.org/abs/2502.20811v1), 2025-02-28

(55) [MV-MATH: Evaluating Multimodal Math Reasoning in Multi-Visual Contexts](http://arxiv.org/abs/2502.20808v1), 2025-02-28

(56) [Digital Player: Evaluating Large Language Models based Human-like Agent in Games](http://arxiv.org/abs/2502.20807v1), 2025-02-28

(57) [Plan2Align: Predictive Planning Based Test-Time Preference Alignment in Paragraph-Level Machine Translation](http://arxiv.org/abs/2502.20795v1), 2025-02-28

(58) [Cyber Defense Reinvented: Large Language Models as Threat Intelligence Copilots](http://arxiv.org/abs/2502.20791v1), 2025-02-28

(59) [Chain-of-Thought Matters: Improving Long-Context Language Models with Reasoning Path Supervision](http://arxiv.org/abs/2502.20790v1), 2025-02-28

(60) [Triple Phase Transitions: Understanding the Learning Dynamics of Large Language Models from a Neuroscience Perspective](http://arxiv.org/abs/2502.20779v1), 2025-02-28

(61) [FlexPrefill: A Context-Aware Sparse Attention Mechanism for Efficient Long-Sequence Inference](http://arxiv.org/abs/2502.20766v1), 2025-02-28

(62) [Collective Reasoning Among LLMs A Framework for Answer Validation Without Ground Truth](http://arxiv.org/abs/2502.20758v1), 2025-02-28

(63) [The Rise of Darkness: Safety-Utility Trade-Offs in Role-Playing Dialogue Agents](http://arxiv.org/abs/2502.20757v1), 2025-02-28

(64) [Mitigating Hallucinations in Large Vision-Language Models by Adaptively Constraining Information Flow](http://arxiv.org/abs/2502.20750v1), 2025-02-28

(65) [Teach-to-Reason with Scoring: Self-Explainable Rationale-Driven Multi-Trait Essay Scoring](http://arxiv.org/abs/2502.20748v1), 2025-02-28

(66) [Measuring Determinism in Large Language Models for Software Code Review](http://arxiv.org/abs/2502.20747v1), 2025-02-28

(67) [SPD: Sync-Point Drop for efficient tensor parallelism of Large Language Models](http://arxiv.org/abs/2502.20727v1), 2025-02-28

(68) [Retrieval Backward Attention without Additional Training: Enhance Embeddings of Large Language Models via Repetition](http://arxiv.org/abs/2502.20726v1), 2025-02-28

(69) [Why Trust in AI May Be Inevitable](http://arxiv.org/abs/2502.20701v1), 2025-02-28

(70) [Towards General Visual-Linguistic Face Forgery Detection(V2)](http://arxiv.org/abs/2502.20698v1), 2025-02-28

(71) [ProAI: Proactive Multi-Agent Conversational AI with Structured Knowledge Base for Psychiatric Diagnosis](http://arxiv.org/abs/2502.20689v1), 2025-02-28

(72) [JAM: Controllable and Responsible Text Generation via Causal Reasoning and Latent Vector Manipulation](http://arxiv.org/abs/2502.20684v1), 2025-02-28

(73) [Prediction of Item Difficulty for Reading Comprehension Items by Creation of Annotated Item Repository](http://arxiv.org/abs/2502.20663v1), 2025-02-28

(74) [Automatic database description generation for Text-to-SQL](http://arxiv.org/abs/2502.20657v1), 2025-02-28

(75) [Consistency Evaluation of News Article Summaries Generated by Large (and Small) Language Models](http://arxiv.org/abs/2502.20647v1), 2025-02-28

(76) [LexRAG: Benchmarking Retrieval-Augmented Generation in Multi-Turn Legal Consultation Conversation](http://arxiv.org/abs/2502.20640v1), 2025-02-28

(77) [Can LLM Assist in the Evaluation of the Quality of Machine Learning Explanations?](http://arxiv.org/abs/2502.20635v1), 2025-02-28

(78) [Are LLMs Ready for Practical Adoption for Assertion Generation?](http://arxiv.org/abs/2502.20633v1), 2025-02-28

(79) [Rectifying Belief Space via Unlearning to Harness LLMs' Reasoning](http://arxiv.org/abs/2502.20620v1), 2025-02-28

(80) [Leveraging Large Language Models for Building Interpretable Rule-Based Data-to-Text Systems](http://arxiv.org/abs/2502.20609v1), 2025-02-28

(81) [NutriGen: Personalized Meal Plan Generator Leveraging Large Language Models to Enhance Dietary and Nutritional Adherence](http://arxiv.org/abs/2502.20601v1), 2025-02-28

(82) [Few-Shot, No Problem: Descriptive Continual Relation Extraction](http://arxiv.org/abs/2502.20596v1), 2025-02-27

(83) [Multi$^2$: Multi-Agent Test-Time Scalable Framework for Multi-Document Processing](http://arxiv.org/abs/2502.20592v1), 2025-02-27

(84) [LLMs Have Rhythm: Fingerprinting Large Language Models Using Inter-Token Times and Network Traffic Analysis](http://arxiv.org/abs/2502.20589v1), 2025-02-27

(85) [Training LLMs with MXFP4](http://arxiv.org/abs/2502.20586v1), 2025-02-27

(86) [ECCOS: Efficient Capability and Cost Coordinated Scheduling for Multi-LLM Serving](http://arxiv.org/abs/2502.20576v1), 2025-02-27

(87) [Visual Reasoning at Urban Intersections: FineTuning GPT-4o for Traffic Conflict Detection](http://arxiv.org/abs/2502.20573v1), 2025-02-27

(88) [Stochastic Rounding for LLM Training: Theory and Practice](http://arxiv.org/abs/2502.20566v1), 2025-02-27

(89) [HuAMR: A Hungarian AMR Parser and Dataset](http://arxiv.org/abs/2502.20552v1), 2025-02-27

(90) [$Q\sharp$: Provably Optimal Distributional RL for LLM Post-Training](http://arxiv.org/abs/2502.20548v1), 2025-02-27

(91) [SoS1: O1 and R1-Like Reasoning LLMs are Sum-of-Square Solvers](http://arxiv.org/abs/2502.20545v1), 2025-02-27

(92) [NANOGPT: A Query-Driven Large Language Model Retrieval-Augmented Generation System for Nanotechnology Research](http://arxiv.org/abs/2502.20541v1), 2025-02-27

(93) [Supervised Fine-Tuning LLMs to Behave as Pedagogical Agents in Programming Education](http://arxiv.org/abs/2502.20527v1), 2025-02-27

(94) [Personas Evolved: Designing Ethical LLM-Based Conversational Agent Personalities](http://arxiv.org/abs/2502.20513v1), 2025-02-27

(95) [CoCa-CXR: Contrastive Captioners Learn Strong Temporal Structures for Chest X-Ray Vision-Language Understanding](http://arxiv.org/abs/2502.20509v1), 2025-02-27

(96) [TripCraft: A Benchmark for Spatio-Temporally Fine Grained Travel Planning](http://arxiv.org/abs/2502.20508v1), 2025-02-27

(97) [A Thousand Words or An Image: Studying the Influence of Persona Modality in Multimodal LLMs](http://arxiv.org/abs/2502.20504v1), 2025-02-27

(98) [Protecting multimodal large language models against misleading visualizations](http://arxiv.org/abs/2502.20503v1), 2025-02-27

(99) [Do Sell-side Analyst Reports Have Investment Value?](http://arxiv.org/abs/2502.20489v1), 2025-02-27

(100) [VideoA11y: Method and Dataset for Accessible Video Description](http://arxiv.org/abs/2502.20480v1), 2025-02-27
